tests/ui/pages/desktop/devhub.py

```diff
@@ -142,12 +142,14 @@ class Header(Region):
         _sign_out_locator = (By.CSS_SELECTOR, '.DevHub-Navigation-SignOut > a')
 
         def click_login(self):
+            self.selenium.find_element_by_id("djHideToolBarButton").click()
```

Comment:
Is that actually needed, does it avoid the other buttons to be clicked? That'd be funny

Reply:
Yeah it is somehow blocking the other buttons.

Reply:
Probably worth adding a comment. Also, in case we disable debug toolbar by default locally in the future, does that work then too or does the "click" fail loudly?

Reply:
It will fail to find the element, and that would be loud.

Reply:
Should be fine I guess :man_shrugging: